ISHPermissionKit/ISHPermissionCategory.h

```diff
@@ -207,6 +207,21 @@ typedef NS_ENUM(NSUInteger, ISHPermissionCategory) {
      *  in the Info PLIST.
      */
     ISHPermissionCategorySpeechRecognition NS_ENUM_AVAILABLE_IOS(10_0) = 10000,
+
+    /**
+     *  Modern way to request permission to local and remote notifications.
+     *
+     *  If your app supports older deployment targets, you can request this
+     *  category before requesting ISHPermissionCategoryNotificationLocal/
+     *  ISHPermissionCategoryNotificationRemote. In that case, users on
+     *  modern systems will only see one permission request if they make
+     *  a decision, but may be prompted twice if they choose "Later".
```

Comment:
Wording should probably reflect that you should make sure to only ask for either of those depending on the user's system.